catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java

```diff
@@ -0,0 +1,363 @@
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(
+                CHEMICAL_BIOLOGICAL_RADIOLOGICAL_NUCLEAR_OPERATION_NAME,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(CHEMICAL_BIOLOGICAL_RADIOLOGICAL_NUCLEAR_TYPE,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(
+                CHEMICAL_BIOLOGICAL_RADIOLOGICAL_NUCLEAR_SUBSTANCE,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(TACTICAL_DATA_LINK_ACTIVITY,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.SHORT_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(TACTICAL_DATA_LINK_TRACK_NUMBER,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(TACTICAL_DATA_LINK_MESSAGE_NUMBER,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(VIDEO_MOTION_IMAGERY_SYSTEMS_MATRIX_LEVEL,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.INTEGER_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(CATEGORY,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+    }
+
+    @Override
+    public Set<AttributeDescriptor> getAttributeDescriptors() {
+        return DESCRIPTORS;
+    }
+
+    @Override
+    public AttributeDescriptor getAttributeDescriptor(String name) {
+        for (AttributeDescriptor attributeDescriptor : DESCRIPTORS) {
+            if (attributeDescriptor.getName()
+                    .equals(name)) {
+                return attributeDescriptor;
+            }
+        }
+        return null;
```

Comment:
It's likely that `getAttributeDescriptor()` will be called rarely.  While these classes implement `MetacardType`, they are used to generate `MetacardType`s and the entire set of attributes will be used.